cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/data/SSTable.java

```diff
@@ -63,7 +63,13 @@ public InputStream openSummaryStream()
     @Nullable
     public InputStream openPrimaryIndexStream()
     {
-        return openInputStream(FileType.INDEX);
+        return openInputStream(isBigFormat() ? FileType.INDEX : FileType.PARTITIONS_INDEX); // Cassandra 4.x vs 5.x
+    }
+
+    @Nullable
+    public InputStream openSecondaryIndexStream()
```

Comment:
How about just call it `openBtiRowIndexStream` to avoid any confusion with secondary index feature in Cassandra?

Reply:
Can we call it `openRowIndexStream`? We did not add `big` or `bti` to open methods.